src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,114 @@
+package duke;
+
+import duke.task.Task;
+import duke.task.Event;
+import duke.task.Todo;
+import duke.task.Deadline;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+import java.time.format.DateTimeFormatter;
+import java.time.LocalDate;
+public class Storage {
+    private final String STORAGE_PATH;
+
+    public Storage(String filePath) {
```

Comment:
Perhaps you can consider using java.nio.Path to specify file path in an OS-independent way. Additionally, you can maybe consider using java.nio.Files to perform write and read functions.

Reply:
I think this tips is very helpful, would like to tryout this method of implementation.